net-p2p/qbittorrent/qbittorrent-5.0.0.ebuild

```diff
@@ -13,45 +13,32 @@ if [[ ${PV} == *9999 ]]; then
 	inherit git-r3
 else
 	SRC_URI="
-		https://downloads.sourceforge.net/qbittorrent/${P}.tar.xz
-		verify-sig? ( https://downloads.sourceforge.net/qbittorrent/${P}.tar.xz.asc )
+		https://downloads.sourceforge.net/project/qbittorrent/${PN}/${P}/${P}.tar.xz
+		verify-sig? ( https://downloads.sourceforge.net/project/qbittorrent/${PN}/${P}/${P}.tar.xz.asc )
 	"
-	KEYWORDS="amd64 ~arm ~arm64 ~ppc64 ~riscv x86"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
 
 	BDEPEND="verify-sig? ( sec-keys/openpgp-keys-qbittorrent )"
 	VERIFY_SIG_OPENPGP_KEY_PATH=/usr/share/openpgp-keys/qBittorrent.asc
 fi
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
 SLOT="0"
-IUSE="+dbus +gui qt6 systemd test webui"
+IUSE="+dbus +gui systemd test webui"
 RESTRICT="!test? ( test )"
-REQUIRED_USE="|| ( gui webui )
+REQUIRED_USE="
+	|| ( gui webui )
 	dbus? ( gui )
 "
 
 RDEPEND="
 	>=dev-libs/openssl-1.1.1:=
 	>=net-libs/libtorrent-rasterbar-1.2.19:=
 	>=sys-libs/zlib-1.2.11
-	qt6? ( >=dev-qt/qtbase-6.2:6[network,ssl,sql,sqlite,xml] )
```

Comment:
Good catch on bumping the minimal versions. It looks like they also bumped the minimal openssl version, not just qtbase and boost.